src/components/tiles/text/text-tile.tsx

```diff
@@ -97,6 +98,9 @@ export default class TextToolComponent extends BaseComponent<ITileProps, IState>
   private textOnFocus: string | string [] | undefined;
   private isHandlingUserChange = false;
 
+  static contextType = LockedContainerContext;
+  declare context: React.ContextType<typeof LockedContainerContext>;
```

Comment:
It would be nice to eventually "modernize" these components so they aren't class based.